assimilation_code/modules/observations/space_quantities_mod.f90

```diff
@@ -0,0 +1,123 @@
+! DART software - Copyright UCAR. This open source software is provided
+! by UCAR, "as is", without charge, subject to all terms of use at
+! http://www.image.ucar.edu/DAReS/DART/DART_download
+!
+
+! ! in this section, define the quantity of interest.  it must
+! ! start with QTY_xxx and be less than 32 characters total.
+! ! if there are units, min/max values, pdf, etc you can add one or
+! ! more name=value pairs.  at this point there *cannot* be spaces
+! ! around the equals sign.  the value can have spaces if it is
+! ! surrounded by double quotes.  e.g. desc="assumes dry air density"
+! ! comment lines (like these) can be added if they start with an additional !
+
+ 
+! BEGIN DART PREPROCESS QUANTITY DEFINITIONS
+!
+! ! kinds for TIEgcm
+!     QTY_ELECTRON_DENSITY
+! 
+! ! kinds for generic parameters that aren't going to be
+! ! directly observed but are going to be adjusted by the
+! ! assimilation.
+!     QTY_1D_PARAMETER
+!     QTY_2D_PARAMETER
+!     QTY_3D_PARAMETER
+! 
+! ! kinds for CHAMP upper atmosphere computations
+!     QTY_ATOMIC_OXYGEN_MIXING_RATIO    
+!     QTY_MOLEC_OXYGEN_MIXING_RATIO     
+! 
+! ! proposed new kinds for COSMIC GPS/RO obs
+! ! (currently unused)
+!     QTY_OCCULTATION_REFRACTIVITY
+!     QTY_OCCULTATION_EXCESSPHASE
+! 
+! ! kind for the other way of measuring elevation
+! ! contrast this with geopotential height
+!     QTY_GEOMETRIC_HEIGHT      units=m
+! 
+! ! kinds for satellite radiances (jason o.)
+!     QTY_INFRARED_RADIANCE
+!     QTY_INFRARED_BRIGHT_TEMP
+!     QTY_LANDMASK
+! 
+! ! kinds for planetary remote sensing (wglawson, c.lee)
+!     QTY_SKIN_TEMPERATURE    units=K
+!     QTY_NADIR_RADIANCE
+!     QTY_TRACER_1_MIXING_RATIO
+!     QTY_TRACER_2_MIXING_RATIO
+!     ! Is QTY_TRACER_MIXING_RATIO necessary with QTY_TRACER_CONCENTRATION
+!     !   also available from the simple advection model?
+!     QTY_SOIL_TEMPERATURE           units="K"     desc="is soil temp really in K or C?"
+!     QTY_SOIL_LIQUID_WATER
+! 
+! ! more kinds for planetary remote sensing (c.lee)
+!     QTY_SURFACE_ALBEDO
+!     QTY_SURFACE_EMISSIVITY
+!     QTY_DUST_OPACITY_7MB
+!     QTY_THC
+! 
+! ! kinds for COAMPS (Tim Whitcomb)
+!     QTY_EXNER_FUNCTION
+!     QTY_TURBULENT_KINETIC_ENERGY
+!     QTY_TOTAL_PRECIPITABLE_WATER
+!     QTY_VERTLEVEL
+!     QTY_MICROWAVE_BRIGHT_TEMP
```

Comment:
coamps is a problem ... models/coamps/external/obs_def/.... has their stuff ... it is an atmospheric model, so at a minimum, these should go in the atmospheric quantities.